buildScript/webpack.config.js

```diff
@@ -44,7 +44,7 @@ export default function makeWebpackConfig(config) {
         html_dir    : 'html',
         use_loader  : true,
         filename    : '[name]-dev.js',
-        deploy_dir  : (process.env.HYDRA_ROOT || '/hydra') + `/server/tomcat/webapps/${config.baseWarName}`,
+        deploy_dir  : (process.env.tomcat_home || '/hydra/server/tomcat') + `/webapps/${config.baseWarName}`,
```

Comment:
This is going to require a change from everyone? right?

Reply:
If you have Tomcat installed at a location other then '/hydra/server/tomcat', you are required to set `tomcat_home` to that location.  Here, using 'tomcat_home' to determine the location of Tomcat's webapps is the right thing to do.  
But, you are right.  I am not sure if this will affect anyone or not.  I am guessing not, but I can't be sure.  I should bring this to everyone's attention at the next standup.

Reply:
Go I know both @ejoliet and I have updated to the most recent version of OSX, which required use to move tomcat.